datafusion/src/physical_plan/sorts/mod.rs

```diff
@@ -32,11 +33,10 @@ use std::borrow::BorrowMut;
 use std::cmp::Ordering;
 use std::fmt::{Debug, Formatter};
 use std::pin::Pin;
+use std::sync::atomic::AtomicUsize;
 use std::sync::{Arc, RwLock};
 use std::task::{Context, Poll};
 
-pub mod external_sort;
```

Comment:
While you removed the `mod` declaration, I think the file is not yet removed in this PR; I suggest

```shell
git rm datafusion/src/physical_plan/sorts/external_sort.rs
```